packages/core/content-manager/admin/src/hooks/useDocumentContext.ts

```diff
@@ -27,6 +27,10 @@ interface DocumentMeta {
    * i.e. { locale: 'fr' }
    */
   params?: Record<string, string | string[] | null>;
+  /**
+   * Field to connect, is the field name to add the newly created relation
+   */
+  fieldToConnect?: string;
```

Comment:
I haven't taken the time to explore this but this is the part I'm a bit uncertain of. There may be a chance to find something more flexible for us in the future

Reply:
yes the part regarding adding the new relation to the parent relation field is the one where I have more doubts because right now when I save a new relation at the same time I update the parent relation using the field name I passed with fieldToConnect and the connect property, we can discuss about it during the real implementation of the feature in the next sprint if we want